secret/variables.tf

```diff
@@ -1,3 +1,9 @@
+variable "access_principals" {
```

Comment:
What do you think about "write" or "edit" instead of "access"? Or even "readwrite"?